src/psyclone/psyGen.py

```diff
@@ -2771,19 +2740,65 @@ def gen_code(self, parent):
 
         :param parent: The parent of this kernel call in the f2pygen AST.
         :type parent: :py:calls:`psyclone.f2pygen.LoopGen`
+
+        :raises NotImplementedError: if there is a name clash preventing to \
```

Comment:
"...clash that prevents the kernel from being module-inlined without ..."

Reply:
Done